### DISCUSSION TITLE
the android studio can not find the Class!

### DISCUSSION THREAD

**Discussion by jayhuang861019** [maplibre/maplibre-native#842] [Q&A] [answered]
i add the dependence to my project like this 
` implementation('org.maplibre.gl:android-sdk:10.0.0'){
        exclude group: 'com.google.android.gms'
}`
then write some code like
`  
LatLng center = map.getCameraPosition().target;
`
it works but the IDE(android studio) can not find the class LatLng 
then i find that the LatLng class in the external libraries is writed in Kotlin,so what should i do to let the IDE find the class?

**Comment by jayhuang861019** [comment]
the class file written in kotlin will not be found by the android studio but the class written by java can be found
[![2023-02-23-09-27-33.png](https://i.postimg.cc/rFWjCDxw/2023-02-23-09-27-33.png)](https://postimg.cc/2VknC80s)
when i open it ,it says like this
![](https://i.postimg.cc/4NJkrRPZ/2023-02-23-09-31-57.png)

**Reply by jayhuang861019** [reply]
what should i do?

**Comment by jayhuang861019** [comment, marked as answer]
ok i got it 
the version of  android studio is too  low

**Reply by louwers** [reply]
Happy you found a solution. 🙂
